cmd/gds/main.go

```diff
@@ -704,11 +704,19 @@ func initClient(c *cli.Context) (err error) {
 		return cli.NewExitError(err, 1)
 	}
 	client = api.NewTRISADirectoryClient(cc)
+	return nil
+}
 
+func initAdminClient(c *cli.Context) (err error) {
```

Comment:
Breaking up the gds client init from the initClient so that connecting to GDS doesn't interfere with connecting to the admin and vice versa.